### PR TITLE
Feature/local file watcher

### PR DESCRIPTION
**Pull request by unhit**
disabled spring jpa
configs accessible as properties classes
services for local and remote file reading (WIP)

### CODE REVIEW

File: src/main/java/com/netps/logreader/dictionary/FileType.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.netps.logreader.dictionary;
+
+public enum FileType {
+    APACHE_LOG,
+    JAVA_LOG
```
Comment (sdudziak) [inline src/main/java/com/netps/logreader/dictionary/FileType.java]: Tu na pewno będzie jeszcze type PHP_LOG jeśli rozróżniamy na technologie

File: src/test/groovy/com/netps/logreader/filewatcher/local/LocalFileWatchServiceTest.groovy [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        def tempFile = File.createTempFile("unittest-", ".tmp")
+        tempFile.deleteOnExit()
+        return tempFile
+    }
+
+    private def recreateFile(File file) {
+        Files.delete(file.toPath())
+        Files.createFile(file.toPath())
+    }
+}
```
Comment (sdudziak) [inline src/test/groovy/com/netps/logreader/filewatcher/local/LocalFileWatchServiceTest.groovy]: I think that also exceptional cases should be added here.

File: src/main/java/com/netps/logreader/dictionary/FileType.java (hunk 1 above)
Comment (unhit) [reply src/main/java/com/netps/logreader/dictionary/FileType.java]: Well, I heard we have to parse the log (or at least extract log level, if applicable).

File: src/main/java/com/netps/logreader/filewatcher/local/LocalFileWatchService.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.input.Tailer;
+import org.apache.commons.io.input.TailerListener;
+
+// TODO exception handler
+@Slf4j
+public class LocalFileWatchService implements FileWatchService {
+
+    private static final long TAILER_DELAY_MS = 1000L; // TODO do cfg
```
[the remark this answers is not in the document]
Comment (unhit) [reply src/main/java/com/netps/logreader/filewatcher/local/LocalFileWatchService.java]: Not with current library. I suggest leaving it as is and implement real-time file watcher in a future from scratch.

**Comment by unhit** [comment]
too many things changed in meantime. another will be opened

**Pull request closed without merging**